client/ayon_core/hosts/maya/plugins/publish/collect_animation.py

```diff
@@ -58,4 +58,3 @@ def process(self, instance):
 
         if instance.data.get("farm"):
             instance.data["families"].append("publish.farm")
-
```

Comment:
I think this may have removed the new line by accident?

Reply:
New line is still in file.